### PR TITLE
Fix profile picture caching

### LINKED ISSUE #7: GitHub API images not getting updated
They show old images, something like caching?
While the link gets updated the images remain same :confused:

### PR DESCRIPTION
**Pull request by mohitkyadav**
Fixes #7

### CODE REVIEW

File: handler.py [hunk 1]
```diff
@@ -36,7 +37,7 @@ def stalk(user):
     count_api_url = os.environ.get('CONTRI_API')
     profile = "​​​​​​​​"
     if api.status_code == 200:
-        pic = "<a href='{0}'>&#8205;</a>".format(res["avatar_url"])
+        pic = "<a href='{0}'>&#8205;</a>".format(res["avatar_url"] + '?' + str(randint(1,100000)))
```
[the remark this answers is not in the document]
Comment (mohitkyadav) [reply handler.py]: I guess that'll work too, timestamp will always be unique.

[the remark this answers is not in the document]
Comment (mohitkyadav) [reply handler.py]: @aashutoshrathi is that supposed to be a "comma".
>res["avatar_url"], timestamp

or "res["avatar_url"] + timestamp"

[the remark this answers is not in the document]
Comment (aashutoshrathi) [reply handler.py]: Yes a `,`.

File: handler.py [hunk 2]
```diff
@@ -36,7 +36,7 @@ def stalk(user):
     count_api_url = os.environ.get('CONTRI_API')
     profile = "​​​​​​​​"
     if api.status_code == 200:
-        pic = "<a href='{0}'>&#8205;</a>".format(res["avatar_url"])
+        pic = "<a href='{0}'>&#8205;</a>".format(res["avatar_url"], datetime.datetime.now().isoformat())
```
[the remark this answers is not in the document]
Comment (mohitkyadav) [reply handler.py]: done

**Comment by aashutoshrathi** [comment]
Looks Perfect :100: 
Deployed your PR, check that out. :smile:

**Pull request merged**